Applications/spire/source/charting/charting_window.cpp

```diff
@@ -226,3 +244,34 @@ void ChartingWindow::on_period_line_edit_changed() {
     m_period_dropdown->set_items({tr("seconds"), tr("minutes"), tr("hours")});
   }
 }
+
+void ChartingWindow::on_security_change(const Security& security) {
+  setWindowTitle(CustomVariantItemDelegate().displayText(
```

Comment:
ChartingWindow doesn't create the model, your test application has to. Propagate this event upward.

Reply:
Should I create the tester then, or put it in main?